announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements-customize.vm

```diff
@@ -19,7 +19,11 @@
 		#end
 	#end
 
-	<div class="page-header"><h3>$tlang.getString("custom.options")</h3></div>
+	<div class="page-header">
+		<h1>
+			$tlang.getString("custom.options")
+		</h1>
```

Comment:
I'd just put this on one line, personally. Unless it really sacrifices readability it makes sense to try and keep the lines down in these templates.